### PR TITLE
[MRG] Fix eeglab test

### LINKED ISSUE #4930: MNE-Python failing test_io_set from mne.io.eeglab.tests.test_eeglab
Hello, I am also getting an error in the same test as issue  #4855 (but a different error, and in  version 0.16) : 

`    
    ================ FAILURES  ================
    _______________    test_io_set   _______________

    args = (), kwargs = {}
    def skipper_func(*args, **kwargs):
        """Skipper for normal test functions."""
        if skip_val():
            raise SkipTest(get_msg(f, msg))
        else:
>           return f(*args, **kwargs)

/user/ngayraud/home/anaconda2/lib/python2.7/site-packages/numpy/testing/decorators.py:147: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
mne/io/eeglab/tests/test_eeglab.py:234: in test_io_set
    np.array([0., 0., 0.]))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

x_id = array([ True,  True,  True], dtype=bool), y_id = array([False, False, False], dtype=bool), hasval = 'nan'

    def chk_same_position(x_id, y_id, hasval='nan'):
        """Handling nan/inf: check that x and y have the nan/inf at the same
            locations."""
        try:
            assert_array_equal(x_id, y_id)
        except AssertionError:
            msg = build_err_msg([x, y],
                                err_msg + '\nx and y %s location mismatch:'
                                % (hasval), verbose=verbose, header=header,
                                names=('x', 'y'), precision=precision)
>           raise AssertionError(msg)
E           AssertionError: 
E           Arrays are not equal
E           
E           x and y nan location mismatch:
E            x: array([ nan,  nan,  nan])
E            y: array([ 0.,  0.,  0.])

/user/ngayraud/home/anaconda2/lib/python2.7/site-packages/numpy/testing/utils.py:704: AssertionError`

I checked the solution proposed in MRG: Fix loc #4860, where all `[0., 0., 0.]` are replaced by `np.nan`.

It seems that the test passes when the line 231-232

`
assert_array_equal(raw.info['chs'][i]['loc'][:3],
                            np.array([0., 0., 0.]))
`

is replaced by

`assert_array_equal(raw.info['chs'][i]['loc'][:3], [np.nan] * 3)`

### PR DESCRIPTION
**Pull request by agramfort**
closes #4930

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -160,7 +160,11 @@ script:
         fi;
         USE_DIRS="";
         for DIR in ${MNE_DIRS}; do
-          USE_DIRS="mne/${DIR}/tests ${USE_DIRS}";
+          if [ "${DIR}" == "." ]; then
+            USE_DIRS="mne/tests ${USE_DIRS}";
```
Comment (larsoner) [inline .travis.yml]: I guess for completeness we might want `mne/*.py` to also catch doc tests on `mne/*.py` files

**Comment by agramfort** [comment]
now doctest is failing on windows 

https://ci.appveyor.com/project/mne-tools/mne-python/build/1.0.1718

@larsoner any suggestion?

**Comment by larsoner** [comment]
If it's a numpy print formatting thing, doctest+SKIP it

**Comment by agramfort** [comment]
how did other projects handle the numpy change of formatting?

**Comment by larsoner** [comment]
I think in SciPy we forced it to use the old printing format because SciPy has its own test runners (it's not bare `pytest`). There is probably something similar we could do, but I'd rather just wait a year or two and switch to the new format.

**Comment by agramfort** [comment]
ok green. merging.

**Pull request merged**